sdk/metricsadvisor/Azure.AI.MetricsAdvisor/src/Models/DataFeed/DataFeedRollupSettings.cs

```diff
@@ -29,19 +32,29 @@ internal DataFeedRollupSettings(DataFeedDetail dataFeedDetail)
         }
 
         /// <summary>
+        /// The value a dimension assumes when it represents a rolled-up value in the data source. Common values
+        /// include <c>__SUM__</c>, <c>NULL</c>, or even an empty string.
```

Comment:
Are these values pre-defined somewhere?

Reply:
Maybe it's not properly explained, but these are user-defined values present in their data source. For instance, if the data source sends the data feed something like this:
| City | Product | Cost |
| --- | --- | --- |
| Redmond | Shoes | 1.00 |
| Redmond | Hat | 2.00 |
| Seattle | Shoes | 3.00 |
| Seattle | Hat | 4.00 |
| __AVG__ | Shoes | 2.00 |
| __AVG__ | Hat | 3.00 |
| Redmond | __AVG__ | 1.50 |
| Seattle | __AVG__ | 3.50 |
| __AVG__ | __AVG__ | 2.50 |

The user already has rolled up data and, when a dimension value is `__AVG__`, it means that that entry has rolled up metrics.

In this case, when creating the data feed, they need to set:
```cs
rollupSettings.RollupType = DataFeedRollupType.AlreadyRollup;
rollupSettings.AlreadyRollupIdentificationValue = "__AVG__";
```

Reply:
The part that wasn't clear is that we metnion potential common values but they don't seem to be part of some pre-defined set. For example, is `__SUM__` an example of an `AlreadyRollupIdentificationValue`?

Reply:
They're not part of a pre-defined set, but are user-specified. It's just an extra piece of info about their data source. I wanted to give some examples to try to make things clear, but it seems it just made things more confusing. 😅 I think it's worth removing that last part.